openpype/plugins/publish/collect_avalon_entities.py

```diff
@@ -59,43 +64,22 @@ def process(self, context):
         task_type = task_info.get("type")
         context.data["taskType"] = task_type
 
-        frame_start = data.get("frameStart")
-        if frame_start is None:
-            frame_start = 1
-            self.log.warning("Missing frame start. Defaulting to 1.")
-
-        frame_end = data.get("frameEnd")
-        if frame_end is None:
-            frame_end = 2
-            self.log.warning("Missing frame end. Defaulting to 2.")
-
-        context.data["frameStart"] = frame_start
-        context.data["frameEnd"] = frame_end
-
-        handles = data.get("handles") or 0
-        handle_start = data.get("handleStart")
-        if handle_start is None:
-            handle_start = handles
-            self.log.info((
-                "Key \"handleStart\" is not set."
-                " Using value from \"handles\" key {}."
-            ).format(handle_start))
-
-        handle_end = data.get("handleEnd")
-        if handle_end is None:
-            handle_end = handles
-            self.log.info((
-                "Key \"handleEnd\" is not set."
-                " Using value from \"handles\" key {}."
-            ).format(handle_end))
-
-        context.data["handles"] = int(handles)
-        context.data["handleStart"] = int(handle_start)
-        context.data["handleEnd"] = int(handle_end)
-
-        frame_start_h = frame_start - context.data["handleStart"]
-        frame_end_h = frame_end + context.data["handleEnd"]
-        context.data["frameStartHandle"] = frame_start_h
-        context.data["frameEndHandle"] = frame_end_h
+        anatomy = context.data["anatomy"]
+
+        frame_info = get_frame_info(asset_entity, anatomy)
+        if frame_info is None:
+            frame_info = UnifiedFrameInfo(1, 2, 0, 0)
+            self.log.warning(
+                "Missing frame information on asset. Using default 1-2 frames."
+            )
```

Comment:
I see that the old logic does the same - but why would it default to 1-2 instead of just a single frame 1-1?

Reply:
To be honest, I don't know.